source/parser.hera

```diff
@@ -4650,7 +4650,7 @@ Catch
     return { $loc, token: $1 }
 
 Class
-  "class" NonIdContinue ->
+  "class" !":" NonIdContinue ->
```

Comment:
1. Should we be consistent about whether colon is excluded from the low-level rule like this, or where it's used (like how you moved Void)?
2. Any chance it makes sense to put colon exclusion in NonIdContinue? Or would that break other things?

Reply:
Ideally they would all be higher up (like in the unary ops).